storage/client_raft_test.go

```diff
@@ -313,7 +313,7 @@ func TestRestoreReplicas(t *testing.T) {
 
 	// Both replicas have a complete list in Desc.Replicas
 	for i, store := range mtc.stores {
-		rng, err := store.GetRange(1)
+		rng, err = store.GetRange(1)
```

Comment:
I think that in cases like this it's probably better to address the shadowing by renaming one of the variables rather than assigning to it. Mutating the outer scope's `rng` variable seems more likely to cause problems than simply allowing the shadowing.